src/scene/axis.js

```diff
@@ -407,19 +417,52 @@ function axisTicksExtend(orient, ticks, oldScale, newScale, size, offset) {
   } else {
     size = {value: sign * size, offset: offset};
   }
+
+  var tickPlacement = config_axis.tickPlacement;
   if (orient === 'top' || orient === 'bottom') {
+    var updatedOldScale = {'scale': oldScale.scale, 'offset': oldScale.offset};
+    var updatedNewScale = {'scale': newScale.scale, 'offset': newScale.offset};
+    if (tickPlacement === 'between' && scaleType === ORDINAL) {
+      var currPadding = 0;
+      var currOuterPadding = 0;
+      for (i=0; i<scales.length; i++) {
+        if (scales[i].name === oldScale.scale) {
+          if ('padding' in scales[i]) {
+           currPadding = scales[i].padding;
+          }
+          if ('outerPadding' in scales[i]) {
+           outerPadding = scales[i].padding;
+          }
+        }
+      }
+      console.log('whattttt');
+      if (!currPadding && typeof config_scale !== 'undefined' && 'padding' in config_scale) {
+        currPadding = config_scale.padding;
+      }
+
+      if (!currOuterPadding && typeof config_scale !== 'undefined' && 'outerPadding' in config_scale) {
+        currOuterPadding = config_scale.outerPadding;
+      }
+
+      var oldPadding = ((oldScale.offset * 2 * currPadding) / ((1 - currPadding) * 2));
+      var newPadding = ((newScale.offset * 2 * currPadding) / ((1 - currPadding) * 2));
+
+      updatedOldScale.offset = ((oldScale.offset - 1) * 2) + oldPadding;
+      updatedNewScale.offset = ((newScale.offset - 1) * 2) + newPadding;
+    }
+
```

Comment:
@arvind This chunk of the code allows the user to set ordinal tick marks to be in between the marks instead of on it.  
For example, this is the default
<img width="186" alt="screen shot 2016-05-20 at 12 49 37 pm" src="https://cloud.githubusercontent.com/assets/5903717/15440258/628a40b6-1e89-11e6-8054-0f82d0f514f2.png">
But this chunk of code allows for 
<img width="183" alt="screen shot 2016-05-20 at 12 52 09 pm" src="https://cloud.githubusercontent.com/assets/5903717/15440320/be12842a-1e89-11e6-8ee1-1a9655f3c24c.png">

In order to figure out the offset of the ticks from the original position, I had to grab the current scale, since the padding and outerPadding gave me the information so I could calculate how far to offset ticks.